src/base/static/js/routes.js

```diff
@@ -156,12 +157,21 @@ var Shareabouts = Shareabouts || {};
       this.appView.newPlace();
     },
 
-    viewLandmark: function(id) {
-      this.appView.viewLandmark(id, { zoom: this.loading });
+    viewLandmark: function(modelId, responseId) {
```

Comment:
I might be missing an easy way to do this, but don't we want to maintain the difference between places and landmarks here in the router, since they have different url signatures? In other words, we want to know when it's appropriate to pass a `datasetSlug` to `viewPlaceOrLandmark` in the app view, right?